harness/determined/common/api/authentication.py

```diff
@@ -81,7 +82,12 @@ def _init_session(
             token = util.get_container_user_token()
 
         if token is not None:
-            return Session(session_user, token)
+            if self.token_store.get_active_user_id() is None:
+                return Session(
+                    session_user, token, _get_current_user_id(self.master_address, token, cert)
```

Comment:
once we fetched it here, should it then be saved back to the token store?

Reply:
thanks, will add